src/docs/library/crosscompiling.md

```diff
@@ -52,17 +52,17 @@ vim Cross.toml
 
 Add the following to the `Cross.toml`. Which will define to `cross` which container should be used for the target architecture:
 
-    ```toml
-    [target.armv7-unknown-linux-gnueabihf]
-    image = "crossimage"
-    ```
+```toml
+[target.armv7-unknown-linux-gnueabihf]
+image = "crossimage"
+```
 
 ```sh
 # Build and tag the container image, specifying the name as defined in the `Cross.toml`.
 docker build -t crossimage crossimage/
 
 # Run cross to compile, cross arguments are the same as the `cargo` ones
-cross build --release --target=armv7-unknown-linux-gnueabihf --features cli
+cross build --path probe-rs-tools --release --target=armv7-unknown-linux-gnueabihf
```

Comment:
@Dirbaio I believe you edited this last time. Can you tell me why the path wasn't specified in the old revision (consequently, can you tell me if the new version is correct?)?

Reply:
I think we had a default that pointed to probe-rs

Reply:
I don't see how that's specified :/

Reply:
I think we are good anyways :)